### DISCUSSION TITLE
zebra for Linux amd64 architecture? (not darwin)

### DISCUSSION THREAD

**Discussion by olivertzeng** [wstyres/Zebra#1957] [Ideas]
zebra for Linux amd64 architecture? (not darwin)

**Comment by nab138** [comment]
What would a use case for this be?

**Comment by olivertzeng** [comment]
Better package manager for Linux, because most GUI package managers sucks
and apps like Cydia Sileo Zebra looks better and seems like it could be
applied to Linux.

nab138 ***@***.***>於 2022年1月24日 週一，00:16寫道：

> What would a use case for this be?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zbrateam/Zebra/discussions/1957#discussioncomment-2027988>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUSZIIL46DXKNXIY55RSRCDUXQSWDANCNFSM5MSX43LA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
曾嘉禾

**Comment by olivertzeng** [comment]
Take that!
https://youtube.com/shorts/zg9eNb1arvE?feature=share
Sileo, on Mac, but I want support for Linux.

曾嘉禾 ***@***.***>於 2022年1月24日 週一，08:34寫道：

> Better package manager for Linux, because most GUI package managers sucks
> and apps like Cydia Sileo Zebra looks better and seems like it could be
> applied to Linux.
>
> nab138 ***@***.***>於 2022年1月24日 週一，00:16寫道：
>
>> What would a use case for this be?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/zbrateam/Zebra/discussions/1957#discussioncomment-2027988>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AUSZIIL46DXKNXIY55RSRCDUXQSWDANCNFSM5MSX43LA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
> --
> 曾嘉禾
>
-- 
曾嘉禾

**Comment by kirb** [comment]
Zebra on Linux would *theoretically* be possible, but it would require rewriting the entire UI layer to use a cross-platform GUI framework like GTK or Qt, or writing a layer that translates iOS UI framework API calls (UIKit) to Linux’s (GTK). The only reason Zebra and Sileo run on macOS is because of Apple’s [Catalyst](https://developer.apple.com/mac-catalyst/), which allows iOS apps to run on macOS. It’s unlikely for the reason that it’s just a ton of work, but still definitely would be cool if it did happen.

**Comment by olivertzeng** [comment]
Envy on Mac OS lol, quite obvious that pkg managers for iOS is way better
than pkg managers on Linux, either Ubuntu software center etc, hope that it
could happen, I’m not the dev, so idk who can I ask to make this happen?

Adam Demasi ***@***.***>於 2022年1月24日 週一，09:59寫道：

> Zebra on Linux would *theoretically* be possible, but it would require
> rewriting the entire UI layer to use a cross-platform GUI framework like
> GTK or Qt, or writing a layer that translates iOS UI framework API calls
> (UIKit) to Linux’s (GTK). The only reason Zebra and Sileo run on macOS is
> because of Apple’s Catalyst <https://developer.apple.com/mac-catalyst/>,
> which allows iOS apps to run on macOS. It’s unlikely for the reason that
> it’s just a ton of work, but still definitely would be cool if it did
> happen.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zbrateam/Zebra/discussions/1957#discussioncomment-2030372>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUSZIIMULF3I27DOZABNPZDUXSXABANCNFSM5MSX43LA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
曾嘉禾

**Comment by olivertzeng** [comment]
Maybe the Linux community? Will they understand all the heck I say about
jailbreaking?

Adam Demasi ***@***.***>於 2022年1月24日 週一，09:59寫道：

> Zebra on Linux would *theoretically* be possible, but it would require
> rewriting the entire UI layer to use a cross-platform GUI framework like
> GTK or Qt, or writing a layer that translates iOS UI framework API calls
> (UIKit) to Linux’s (GTK). The only reason Zebra and Sileo run on macOS is
> because of Apple’s Catalyst <https://developer.apple.com/mac-catalyst/>,
> which allows iOS apps to run on macOS. It’s unlikely for the reason that
> it’s just a ton of work, but still definitely would be cool if it did
> happen.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zbrateam/Zebra/discussions/1957#discussioncomment-2030372>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUSZIIMULF3I27DOZABNPZDUXSXABANCNFSM5MSX43LA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
曾嘉禾

**Comment by olivertzeng** [comment]
Still, I hope that you could make one (Zebra for Linux amd64 architecture)

Adam Demasi ***@***.***>於 2022年1月24日 週一，09:59寫道：

> Zebra on Linux would *theoretically* be possible, but it would require
> rewriting the entire UI layer to use a cross-platform GUI framework like
> GTK or Qt, or writing a layer that translates iOS UI framework API calls
> (UIKit) to Linux’s (GTK). The only reason Zebra and Sileo run on macOS is
> because of Apple’s Catalyst <https://developer.apple.com/mac-catalyst/>,
> which allows iOS apps to run on macOS. It’s unlikely for the reason that
> it’s just a ton of work, but still definitely would be cool if it did
> happen.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zbrateam/Zebra/discussions/1957#discussioncomment-2030372>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUSZIIMULF3I27DOZABNPZDUXSXABANCNFSM5MSX43LA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
曾嘉禾
